session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -747,14 +747,37 @@ public void deleteData(String path, long time)
    * delete data <= time in multiple timeseries
    *
    * @param paths data in which time series to delete
-   * @param time  data with time stamp less than or equal to time will be deleted
+   * @param endTime data with time stamp less than or equal to time will be deleted
    */
-  public void deleteData(List<String> paths, long time)
+  public void deleteData(List<String> paths, long endTime)
       throws IoTDBConnectionException, StatementExecutionException {
     TSDeleteDataReq request = new TSDeleteDataReq();
     request.setSessionId(sessionId);
     request.setPaths(paths);
-    request.setTimestamp(time);
+    request.setStartTime(Long.MIN_VALUE);
+    request.setEndTime(endTime);
+
```

Comment:
It may simply be changed to a call of `deleteData(paths, Long.MIN_VALUE, endtime)`.

Reply:
Fixed